try:
    print(x)
except:
    print('err')